distributed/worker.py

```diff
@@ -2046,7 +2046,9 @@ async def gather_dep(self, worker, dep, deps, total_nbytes, cause=None):
                     )
 
                 total_bytes = sum(
-                    self.tasks[key].nbytes or 0 for key in response["data"]
+                    self.tasks[key].nbytes or 0
```

Comment:
@gforsyth what about this?

```suggestion
                    self.tasks[key].get_nbytes()
```

Reply:
Heh, that _would_ make a great deal of sense